aspnetcore/toc.md

```diff
@@ -379,11 +388,12 @@
 #### [Key immutability and settings](xref:security/data-protection/implementation/key-immutability)
 #### [Key storage format](xref:security/data-protection/implementation/key-storage-format)
 #### [Ephemeral data protection providers](xref:security/data-protection/implementation/key-storage-ephemeral)
-### [Compatibility](xref:security/data-protection/compatibility/index)
+### Compatibility
+#### [Overview](xref:security/data-protection/compatibility/index)
 #### [Replace <machineKey> in ASP.NET](xref:security/data-protection/compatibility/replacing-machinekey)
+## [Safe storage of app secrets in development](xref:security/app-secrets)
```

Comment:
The word "Safe" doesn't provide much value here, IMO.

Reply:
Secure? Historically dev secrets weren't stored securely.

Reply:
"Secure" sounds too strong, I'm inclined to keep "Safe".

Reply:
Or truncate to "Storage of secrets in development"